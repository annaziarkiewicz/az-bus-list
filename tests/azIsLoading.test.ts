import { describe, test, expect } from "vitest";
import { render } from "@testing-library/vue";

import AzIsLoading from "@/components/azIsLoading.vue";

describe("AzIsLoading", () => {
  test("it should match snapshot", () => {
    // Act
    const { container } = render(AzIsLoading);

    // Assert
    expect(container).toMatchSnapshot();
  });
});
