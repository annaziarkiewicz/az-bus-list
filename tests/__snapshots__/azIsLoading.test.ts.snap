// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AzIsLoading > it should match snapshot 1`] = `
<div>
  <div
    class="az-is-loading"
    data-v-9a939c95=""
  />
</div>
`;
